android/crt/build.gradle

```diff
@@ -53,7 +52,8 @@ android {
 
     defaultConfig {
         minSdkVersion 24  // TODO - dictated by CompletableFuture which is API 24+
-        targetSdkVersion 30
+        targetSdkVersion 33
```

Comment:
It was complaining that targetSDK 30 is not supported, and the minimum version should be 33 if you want to submit apps to Google Play. While we don't need to submit it to Google Play, 33 seems to work.

Reply:
We are not updating the minSdkVersion, so we should be fine.